src/content/1.7/basics/keeping-up-to-date/migration.md

```diff
@@ -455,7 +455,7 @@ by your server configuration. If uploading a single CSV file is
 impossible, split it in half and try again.
 
 You have to go through the configuration for each import, which can be
-tedious in some cases. For instance, if the provided columns do not
+tedious in some cases. For instance, if the provided columns does not
```

Comment:
I'm not sure about that, IFAIK:
- a column does
- column**s** do

Reply:
Agreed with @atomiix

Reply:
> 'Does' is only used in singular, third person. In other cases, 'do' is used. 

https://www.dropbox.com/s/mmmgynbla0ysu3w/Zrzut%20ekranu%202021-06-4%20o%2012.41.37.png?dl=0